charts/microgateway/.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+
+tests/__snapshot__/
```

Comment:
This folder and the files contained within are used in the unit test configmap_test.yaml as a reference for correct output. They should not be deleted.
The DEVELOPER_NOTES.md contains information on using / updating the snapshots.